src/AdminApp/components/RestaurantList/RestaurantView/RestaurantView.jsx

```diff
@@ -117,6 +118,7 @@ const RestaurantView = ({
           },
         })
        setEditableDetails(savedDetails);
```

Comment:
Kavitha, Use effect is not triggered before , now refetch will solve this issue and trigger useffect , Tested in DB, as wel as UI.